test/Organizer/OrganizerLDProjectorTest.php

```diff
@@ -593,10 +642,17 @@ private function mockGet($organizerId, $fileName)
      */
     private function expectSave($organizerId, $fileName)
     {
-        $organizerWithLabelJson = file_get_contents(__DIR__ . '/Samples/' . $fileName);
+        $expectedOrganizerJson = file_get_contents(__DIR__ . '/Samples/' . $fileName);
+        // The expected organizer json still has newline formatting.
+        // The actual organizer json on the other hand has no newlines
+        // because it was created by using the withBody method on JsonDocument.
+        // By calling json_encode(json_decode(...)) the newlines are also removed
+        // from the expected document.
+        $expectedOrganizerJson = json_encode(json_decode($expectedOrganizerJson));
```

Comment:
Goeie workaround. Ik heb vaak dezelfde issue met PHPStorm die een newline achter de test data smijt. Meestal open ik de file met de expected json in iets zoals Textmate en verwijder de trailing newline.